universal-application-tool-0.0.1/app/services/applicant/ApplicantQuestion.java

```diff
@@ -582,6 +592,71 @@ public Path getNumberPath() {
     }
   }
 
+  public class MultiOptionQuestion implements PresentsErrors {
```

Comment:
Question (blocking): MultiOptionQuestionDefinition is an abstract class while this class is concrete. Is the plan to have all multi-option questions to use this class, or did you mean DropdownQuestion here?

Reply:
Great question - I believe all multi-option questions can use this class, at least for now. The ApplicantQuestion class provides methods for getting the applicant's answer if it exists, checking for validation errors, etc. - all of which at the moment are the same across multi-option questions.

We may want to have two different ones in the future - one for multi-answer questions (list) and one for single-answer questions (single string)

Reply:
Done! I changed the class name to `SingleSelectQuestion` and added a TODO comment for checkboxes (i.e. `MultiSelectQuestion`)